std_msgs/Header header
float32 x
float32 y
float32 velocity_x
float32 velocity_y
float32 dx
float32 dy
float32 accel_x
float32 accel_y
float32 yaw #radian
float32 yawrate #rad/s
float32 steering_angle #radian